sydent/sydent.py

```diff
@@ -40,10 +42,13 @@
 from http.servlets.bulklookupservlet import BulkLookupServlet
 from http.servlets.pubkeyservlets import Ed25519Servlet
 from http.servlets.threepidbindservlet import ThreePidBindServlet
+from http.servlets.threepidunbindservlet import ThreePidUnbindServlet
 from http.servlets.replication import ReplicationPushServlet
 from http.servlets.getvalidated3pidservlet import GetValidated3pidServlet
 from http.servlets.store_invite_servlet import StoreInviteServlet
 
+from db.version import VersionStore
```

Comment:
shouldn't this be up with "from db.sqlitedb import SqliteDatabase" ?

Reply:
It's gone now anyway